docs/events/messages/product_updated.yaml

```diff
@@ -0,0 +1,66 @@
+product_updated:
+  title: Details of the product that has been updated
+  payload:
+    type: object
+    properties:
+      timestamp:
+        description: The Unix epoch time when this update took place
+        type: integer
+        examples:
+          - 1234567890
+      code:
+        description: The code of the product that was updated
+        type: string
+        examples:
+          - "3017620420009"
+      rev:
+        description: The revision number for the change
+        type: integer
+        examples:
+          - 194
+      product_type:
+        description: The high-level type product, e.g. "food", "beauty", "petfood"
+        enum:
+          - food
+          - beauty
+          - petfood
+          - product
+      flavor:
+        deprecated: true
+        description: |
+          DEPRECATED: The product_type field should be used to identify the kind of product this event relates to.
+          The Product Opener application type that generated the message, e.g. off, opff, off-pro
+        enum:
+          - off
+          - opff
+          - obf
+          - opf
+          - off-pro
+      user_id:
+        description: The public facing user id of the user that made the edit
+        type: string
+      action:
+        description: The nature of the change, e.g. "created", "deleted"
+        enum:
+          - updated
+          - created
+          - deleted
+          - archived
+          - unarchived
+          - reprocessed
+      comment:
+        description: Overview of the change, provided by the user
+        type: string
+      diffs:
```

Comment:
The diffs brings a lot of information (we use it in Robotoff, Open Prices, and soon in Nutripatrol), so I think it's worth it to document it.
I've spent a bit of time on it, so I can document it once this PR is merged :)

Reply:
Hi @raphael0202 , I've added a bit more detail. Is that enough?